src/styles/_variables.scss

```diff
@@ -24,25 +24,27 @@ $LH7: 1;
 $LH8: 0.5;
 
 $M1: #fff;
-$M2: #fafafa;
-$M3: #30323e;
-$M4: #606060;
+$M2: #F7F8F9;
+$M3: #3E3E3E;
+// set new M3 and new M4 to between old B3 and old M4
```

Comment:
If designer determine to remove `$M4` from this project, please search global and replace `$M4` with `$M3`, we don't need to assign `$M4` to `$M3`

Reply:
This update is a preview edition, maybe we'll use the $M4 in the future, I will remove it when the design is stable edition.